### ISSUE TITLE
Exhibition equipment sourcing

### ISSUE THREAD

**Issue by ana0** [ournetworks/2019#97]
Minimum viable exhibition equipment:

- [x] wired ethernet (Brian Franklin)
- [x] waist high plinth (Brian Franklin)
- [x] HD video projector with media playing capability (Farhad Bahram)
- [x] stand or mount for projector (Farhad Bahram)
- [x] two projectors with pluggable media playing (Tobias Williams)
- [x] stand or mount for projectors (Tobias Williams)
- [x] ability to wall mount backlit poster (Nahee Kim)
- [x] a projector or monitor (Biyi Wen)
- [x] a mouse (Biyi Wen)
- [x] a computer to run the website (Biyi Wen)
- [x] HDMI connecting cable (Biyi Wen)
- [x] stand for monitor or projector  (Biyi Wen)

**Comment by dcwalk** [comment]
Here is where we've documented borrowed A/V (inc. from TMAC Gamma Space and Charles St.): #33 

But additionally, and we should maybe request the final list from them, I requested a 4K projecter from Charles Street Video for the park screening.

This is the description of equipment that Jennie sent:

> For the equipment, I have rented 3 70” screens from CSV. They are $30 per day or $75/week, so the weekly rate makes the most sense. The other equipment (microphones, lectern, mixer, speakers) belongs to DMG and Gamma so we can loan those to you without charge. 

> CSV has a super-bright 4K projector that is $50/day, and a number of less-bright options that work perfectly well in the Gamma Gallery since there are no windows. Those are $25/day.

We worked venue costs incl. this into the budget, so not sure how the above fares w/r/t renting :smile:

**Comment by ana0** [comment]
Ok, the thread is a little hard to parse because it's pretty long. It sounds like we're good for projectors and screens for the kickoff (#95) and screening, but may need 2-3 more of the low-brightness projectors for the exhibition. 

Ben committed to bringing one, and Tobias is ok with single channel if needed, so that leaves us with 2 at minimum. I think some kind of master list would be helpful .. what's the best way to coordinate it?

**Comment by dcwalk** [comment]
We have the ultimate equipment list here: https://docs.google.com/spreadsheets/d/1g32Cu8Gl-E6eb0UM5JMNuMANKJK-yfSBPDtzjr_l24I/edit#gid=0

**Comment by dcwalk** [comment]
We sourced things! closing
